install/debian/preinst

```diff
@@ -8,11 +8,21 @@ case "$1" in
     # Old v7 init script is broken and does not stop when told so
     if [ -e /etc/init.d/FAHClient ]; then
       systemctl kill FAHClient 2>/dev/null || true
+
+      # Missing PID file in init scripts cripples systemd monitoring,
+      # requiring manual waiting for process exit
+      TRY=0
+      while pgrep -x FAHClient >/dev/null && [ $TRY -lt 120 ]; do
+        sleep 1
+        TRY=$((TRY+1))
+      done
```

Comment:
This could be replaced by

    timeout 120 pidwait -x FAHClient

``timeout`` is part of ``coreutils`` and ``pidwait`` is from ``procps``.

Reply:
We cannot rely on `pidwait`, as it is not available on all (still supported) distributions, including Ubuntu 20.04 and RHEL 8.

Reply:
indeed